docs/api.md

```diff
@@ -1361,7 +1361,7 @@ most.from([1, 2, 3, 4])
 ### switch
 
 ####`stream.switch() -> Stream`
-####`most.switch(stream) -> Stream`
+####`most.switchLatest(stream) -> Stream`
```

Comment:
I'm in favor of `switchLatest` only. We should try to avoid aliases as it leads to confusion when one person uses one name and other uses the other.

In this same context I think we should consider removing other aliases like `flatMapEnd/continueWith` and `flatMapError/recoverWith`,  `flatMap/chain`, `until/since`, and probably some others. We should look to have only 1 way to correctly do some desired behavior.